stroeer/web/section/v1/web_section_page.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package stroeer.web.section.v1;
+
+import "stroeer/web/article/v1/web_article.proto";
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/web/section/v1;section";
+option java_multiple_files = true;
+option java_package = "de.stroeer.web.section.v1";
+
+message SectionPageStage {
+  repeated stroeer.web.article.v1.WebArticle web_articles = 1;
+}
+
+message SectionPageBox {
```

Comment:
First of all: I don't like the name.

I would add a new `proto` file for this "feature" because it is shared on article and section pages.

Name proposals:
- this
- is
- fucking
- hard
- :(
- recommendations

We should ask the whole team.